test/test_jujuversion.py

```diff
@@ -60,6 +60,10 @@ def test_has_app_data(self):
         self.assertTrue(JujuVersion('2.7.0').has_app_data())
         self.assertFalse(JujuVersion('2.6.9').has_app_data())
 
+    def test_is_dispatch_aware(self):
+        self.assertTrue(JujuVersion('2.8.0').is_dispatch_aware())
+        self.assertFalse(JujuVersion('2.7.9').is_dispatch_aware())
```

Comment:
I would also check that "2.10.0 is aware", as it's what would fail if you compare in the wrong way.

Reply:
that sounds like a check that 2.10 comes after 2.8. There isn't an explicit check for that, so I added it.